acceptance/main.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Peter Mattis (peter.mattis@gmail.com)
+
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+	"os/signal"
+	"regexp"
+	"sort"
+	"strings"
+	"time"
+)
+
+type stopper chan struct{}
+
+var numNodes = flag.Int("n", 3, "the number of nodes to start (if not otherwise specified by a test)")
+var repeat = flag.Int("r", 1, "the number of times to run each test")
+var tests = map[string]func(stopper){}
+
+func registerTest(name string, f func(stopper)) int {
```

Comment:
Can we just use `go test` here instead of building our own registration and running system? (maybe put the stopper in a global variable instead of passing it as an argument)